spec/lib/service_provider_spec.rb

```diff
@@ -211,10 +212,10 @@
 
   describe '#block_encryption' do
     context 'when no value is specified in YAML' do
-      it 'returns "none"' do
```

Comment:
I actually like hardcoding things like this in the tests so that the test enforces the policy and it makes changing the default a more conscious and detectable thing.  (maybe thats just me though).